### DISCUSSION TITLE
LocalDevEnv.ps1 with own Tenant Bacpac

### DISCUSSION THREAD

**Discussion by MUR-SDP** [microsoft/AL-Go#911] [Q&A] [answered]
Hello...   Is there an easy way to use the LocalDevEnv.ps1 script and mount our own tenant .bacpac database backup instead of the standard Cronus db?

I know it can be done with the BCContainerHelper but then that will have to be maintained outside of AL-Go. :)

Thanks so much...

**Comment by freddydk** [comment, marked as answer]
The only way to do this is by overriding the NewBcContainer mechanism as described loosely here: https://github.com/microsoft/AL-Go/blob/main/Scenarios/settings.md#run-alpipeline-script-override

**Reply by MUR-SDP** [reply]
Thanks Freddy... appreciate the prompt reply.
